components/analytics/components/Filter/index.js

```diff
@@ -11,30 +11,64 @@ import { isValidPlan } from '../../../../utils';
 
 const { Option } = Select;
 
-const dataRangeFilters = {
-	monthly: {
-		from: moment()
-			.subtract(30, 'days')
-			.format('YYYY/MM/DD'),
+const dateRangeFilters = {
+	'This week': {
+		from: moment().subtract(7, 'days').format('YYYY/MM/DD'),
```

Comment:
@jyash97 That is wrong too.
It should be {start of the current week} to {current date}